### ISSUE TITLE
Target call number (item) data when calculating Key Collections (Diss and JSTOR)

### ISSUE THREAD

**Issue by ryan-jacobs** [Center-for-Research-Libraries/vufind#51]
JSTOR will eventually be a Key Collection facet term but the marc field pattern value needed to identify it is not yet clear.

We will identify JSTOR call numbers at the item level, but how that maps to the iventory record marc data that VuFind can harvest over OAI it not yet clear. We will need some clarification from EBSCO before we can move that forward.

See also #47

**Comment by ryan-jacobs** [comment]
This is on-hold until it's clear how call numbers will propagate from item records to the marc VuFind has access to.

**Comment by ryan-jacobs** [comment]
Jstor item date should be rolling in now. We can also see that the OAI connection is computing a 952 value for all item records at harvest time, and the 952e contains the call number. After gathering some info from subject matter experts, including a detailed discussion with @AndyElliottCRL, we concluded that we should use the 952e as the primary reference for call number data but also have a fallback to the 099a for cases where item records may not have been developed yet:

- **Dissertations**: check 952e (multiple) for "P-", also fallback on check of 099a (single) for "P-"
- **JSTOR**: check 952e (multiple) for "J-", also fallback on check for 099a (single) for "JSTOR"

**Comment by ryan-jacobs** [comment]
Moving back to on hold. The jstor items are still pending load.

**Comment by ryan-jacobs** [comment]
In the meantime I am going to add some temporary handling for JSTOR key collection support based on the 099. This will allow most of the collection to be built, but will not be the permanent solution.
